src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.operation;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void execute(Map.Entry<String, Integer> entry, int value) {
+        entry.setValue(value);
+    }
+
+    public static class ReturnOperation implements OperationHandler {
```

Comment:
The `ReturnOperation` class should be placed in a separate file within the 'strategy' package to follow the Strategy pattern and adhere to the checklist.